plugins/com.google.cloud.tools.eclipse.ui.util/src/com/google/cloud/tools/eclipse/ui/util/images/SharedImages.java

```diff
@@ -40,9 +34,7 @@
    * @return the image or null if the image was not found
    */
   public static Image createRefreshIcon(Device device) {
```

Comment:
This class is … a bit unconventional :-)  The SWT convention is that the creator of an image is responsible for disposing of it.  In this case, this class is creating the image, and the panel is just using it.  Conventions can be broken of course.  The `org.eclipse.ui.IWorkbenchImages` provides `ImageDescriptor`s; it also provides `Image` instances from the central workbench image registry, which are good for the lifetime of the workbench.

This doesn't need to be changed in this PR.